apps/docs/pages/guides/auth/auth-helpers/sveltekit.mdx

```diff
@@ -9,29 +9,75 @@ export const meta = {
 
 This submodule provides convenience helpers for implementing user authentication in [SvelteKit](https://kit.svelte.dev/) applications.
 
-## Installation
+## Configuration
+
+### Install SvelteKit Auth Helpers library
 
 This library supports Node.js `^16.15.0`.
 
 ```sh
 npm install @supabase/auth-helpers-sveltekit
 ```
 
-## Getting Started
-
-### Configuration
+### Declare Environment Variables
 
-Set up the following env vars. For local development you can set them in a `.env` file. See an [example](https://github.com/supabase/auth-helpers/blob/main/examples/sveltekit/.env.example).
+Retrieve your project's URL and anon key from your [API settings](https://app.supabase.com/project/_/settings/api), and create a `.env.local` file with the following environment variables:
 
-```bash
+```bash title=".env.local"
 # Find these in your Supabase project settings https://app.supabase.com/project/_/settings/api
 PUBLIC_SUPABASE_URL=https://your-project.supabase.co
 PUBLIC_SUPABASE_ANON_KEY=your-anon-key
 ```
 
-### Set up the Supabase client
+### Creating a Supabase Client
+
+<Tabs
+  scrollable
+  size="small"
+  type="underlined"
+  defaultActiveId="js"
+>
+<TabPanel id="js" label="JavaScript">
 
-Create a server supabase client in a handle hook:
+Create a new `hooks.server.js` file in the root of your project and populate with the following:
+
+```js title=src/hooks.server.js
```

Comment:
Just a heads up, when the [code hike](https://github.com/supabase/supabase/pull/14787) switch merges these will just be:
```markdown
```js src/hooks.server.js
...
```
(no `title=`)

Reply:
Ah ok let me change those before merging this PR.

Reply:
https://github.com/supabase/supabase/pull/14787 merged now 👍